cargo install broken on latest release

rustc & cargo version is 1.65.0

```
$ cargo install mdbook-d2
....
   Compiling toml v0.5.11
   Compiling mdbook-d2 v0.2.0
error[E0277]: the trait bound `Value: From<toml::map::Map<std::string::String, toml::value::Value>>` is not satisfied
  --> /Users/patrickpilch/.cargo/registry/src/github.com-1ecc6299db9ec823/mdbook-d2-0.2.0/src/backend.rs:69:34
   |
69 |         let value: toml::Value = ctx.config.get_preprocessor("d2").unwrap().clone().into();
   |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ---- required by a bound introduced by this call
   |                                  |
   |                                  the trait `From<toml::map::Map<std::string::String, toml::value::Value>>` is not implemented for `Value`
   |
   = help: the following other types implement trait `From<T>`:
             <Value as From<&'a str>>
             <Value as From<BTreeMap<S, V>>>
             <Value as From<HashMap<S, V>>>
             <Value as From<Vec<V>>>
             <Value as From<bool>>
             <Value as From<f32>>
             <Value as From<f64>>
             <Value as From<i32>>
           and 7 others
   = note: required for `toml::map::Map<std::string::String, toml::value::Value>` to implement `Into<Value>`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `mdbook-d2` due to previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "mdbook-d2"
 description = "D2 diagram generator plugin for MdBook"
-version = "0.2.0"
+version = "0.2.1"
 authors = ["Daniel Eades <danieleades@hotmail.com>"]
 edition = "2021"
 license = "MIT"
@@ -18,7 +18,7 @@ mdbook = "0.4"
 pulldown-cmark = "0.9"
 pulldown-cmark-to-cmark = "10.0"
 serde = { version = "1.0.152", features = ["derive"] }
-toml = "0.5.10"
+toml = "0.7"
 anyhow = "1.0.69"
 
 [dev-dependencies]

--- a/README.md
+++ b/README.md
@@ -12,13 +12,13 @@ A preprocessor for [mdbook](https://github.com/rust-lang/mdBook) to convert
 Install with cargo:
 
 ```sh
-cargo install mdbook-d2
+cargo install mdbook-d2 --locked
 ```
 
 Or to install from git:
 
 ```sh
-cargo install --git https://github.com/danieleades/mdbook-d2
+cargo install --git https://github.com/danieleades/mdbook-d2 --locked
 ```
 
 ## Requirements

